src/app/home/avatar-generator/AvatarSelectorPath.ts

```diff
@@ -0,0 +1,373 @@
+export function getHair(type: string, color: string): string {
```

Comment:
Please update the name of this file to use kebab-case, to keep it consistent with the rest of the files.